ietf/home/templates/home/iab_home_page.html

```diff
@@ -1,58 +1,77 @@
-{% extends settings.utils.LayoutSettings.base_template %}
-{% load static wagtailimages_tags %}
+<!-- Prettier will move sequential wagtail tags onto one line, which stops them working. Use the comment below to stop it reformatting them. -->
+<!-- prettier-ignore -->
```

Comment:
Maybe you could add a `.prettierignore` file with the contents:
```
*.html
```

https://prettier.io/docs/en/ignore.html

Reply:
(that's been the approach on some other projects)

Reply:
There is one already, to stop prettier from breaking wagtail sitewide.

The line-by-line ignores allow me to run prettier locally and get nice indentation without breaking wagtail. It might not be everyone's preferred solution but it works for me.